hoodie-common/src/main/java/com/uber/hoodie/common/table/log/HoodieCompactedLogRecordScanner.java

```diff
@@ -167,17 +176,48 @@ private void loadRecordsFromBlock(
   }
 
   /**
+   * Get Class from the payload fully qualified class name
+   *
+   * @param className
+   * @return
+   */
+  private Class<? extends HoodieRecordPayload> getClazz(String className) {
```

Comment:
Can you double check if this can already be done via some `ReflectionUtils` class? (you can just add specific try-catch blocks then)

Reply:
will check.